apps/src/applab/applab.js

```diff
@@ -666,6 +666,9 @@ Applab.init = function(config) {
   studioApp.runButtonClick = this.runButtonClick.bind(this);
 
   Applab.channelId = config.channel;
+  if (config.level.testAssetPathPrefix) {
+    Applab.assetPathPrefix = config.level.testAssetPathPrefix;
+  }
```

Comment:
Are we actually getting coverage of the default asset path prefix? I would have guessed that in our tests we need to localhost:8001 for things to work, and thus that anywhere this actually gets used we end up overriding it anyways.